neuron/src/app/Neuron/CLI/Search.hs

```diff
@@ -23,8 +23,8 @@ searchScriptArgs :: SearchCommand -> [String]
 searchScriptArgs SearchCommand {..} =
   let searchByArgs =
         case searchBy of
-          SearchByTitle -> ["(^# )|(^title: )", "2"]
-          SearchByContent -> ["", "2"]
+          SearchByTitle -> ["(^# )|(^title: )", "2", "/*.{md,org}"]
```

Comment:
By the way, another issue: the `^#` here won't work for org files, which use `*` prefix for the title. You will want to test the change on actual org files with specific titles, and make sure that searching for the title matches them.

Reply:
Sorry about that I'm new to the `.org`. I'll look into it.